package.json

```diff
@@ -2,9 +2,11 @@
   "name": "waetherjs",
   "version": "1.0.0",
   "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
-  "main": "main.js",
+  "main": "index.js",
+  "private": "true",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
Kindly remove the "test" script and replace it with the "dev" script, so we can run both the development and production scripts :+1: